Space-Zoologist/Assets/Scripts/Dialogue/DialogueParser.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using DialogueEditor;
+
+public class DialogueParser : MonoBehaviour
```

Comment:
How is this script being used right now? It seems a bit unclear at the moment.

Reply:
This is a parser that helps parse a document like this
https://drive.google.com/drive/u/1/folders/1K4zo5WapP2cGvuXvnIGQPURrI3ceBhZb
and turn it into dialogue nodes (basically, it's helps with making dialogue).

This script is not meant to be used in gameplay. It's a one-time script that creates a dialogue object that can be moved to the dialogue manager.